crux-mir/src/Mir/Generate.hs

```diff
@@ -77,23 +77,26 @@ compileMirJson keepRlib quiet rustFile = do
             [ "--cfg", "crux", "--cfg", "crux_top_level"
             , "-Z", "ub-checks=false"
             , "-o", outFile]
-    let cp' = if not quiet then cp else
-            (cp { Proc.std_out = Proc.NoStream, Proc.std_err = Proc.NoStream })
-    ec <- Proc.withCreateProcess cp' $ \_ _ _ ph -> Proc.waitForProcess ph
+    (ec, sout, serr) <- Proc.readCreateProcessWithExitCode cp ""
     case ec of
-        ExitFailure cd -> fail $
-            "Error " ++ show cd ++ " while running mir-json on " ++ rustFile
+        ExitFailure cd -> fail $ unlines $
```

Comment:
Embedding this output in `fail` seems to subject it to `crux-mir`'s colorization, which feels like a bummer, given how richly colored `rustc` errors are. Local experimentation suggests we could avoid this if we
- add `--color=always` to the `mir-json` invocation,
- `hPutStrLn stderr` this message, and
- fail out afterwards with `fail "compilation error"` or some such simple message.

Does that sound plausible and worthwhile to you?